tests/test_theming/test_templating.py

```diff
@@ -23,19 +23,26 @@ def test_autosummary_class_template_overloading(make_app, app_params):
     setup_documenters(app)
     app.build()
 
-    result = (app.outdir / 'generated' / 'sphinx.application.TemplateBridge.html').read_text(encoding='utf8')
+    result = (app.outdir / 'generated' / 'sphinx.application.TemplateBridge.html').read_text(
```

Comment:
just wondering, but the format is because of `ruff format` and you exceeded the line length?

Reply:
yep that is correct, just a little to long I believe